test/src/test/testtool/ManagedFileHandler/scenario01.properties

```diff
@@ -3,16 +3,18 @@ scenario.description = MFH process one xml file
 include = common.properties
 
 dir.init.filename						= scenario01/dir-init.xml
+dir.moveFile.filename					= scenario01/move-file.xml
 
 step1.jdbc.deleteRecords.read			= common/rowsupdated-dummy.xml
 
 step2.dir.init.writeline				= dummy
+step3.dir.moveFile.writeline			= dummy
 
-step3.delay.process.writeline			= dummy
-step4.delay.process.readline			= dummy
+step4.delay.process.writeline			= dummy
+step5.delay.process.read    			= ignore
```

Comment:
Changing this from `readline` to `read` allows the scenario to proceed until the end.